spec/models/validations/household_validations_spec.rb

```diff
@@ -651,24 +695,30 @@
           .to be_empty
       end
 
-      it "prevten cannot be 3" do
-        record.referral = 1
-        record.prevten = 3
-        household_validator.validate_previous_housing_situation(record)
-        expect(record.errors["prevten"])
-          .to include(match I18n.t("validations.household.prevten.internal_transfer", prevten: ""))
-        expect(record.errors["referral"])
-          .to include(match I18n.t("validations.household.referral.prevten_invalid", prevten: ""))
-      end
-
-      it "prevten cannot be 4, 10, 13, 19, 23, 24, 25, 26, 28, 29" do
-        record.referral = 1
-        record.prevten = 4
-        household_validator.validate_previous_housing_situation(record)
-        expect(record.errors["prevten"])
-          .to include(match I18n.t("validations.household.prevten.internal_transfer", prevten: ""))
-        expect(record.errors["referral"])
-          .to include(match I18n.t("validations.household.referral.prevten_invalid", prevten: ""))
+      [
+        { code: 3, label: "Private sector tenancy" },
+        { code: 4, label: "Tied housing or rented with job" },
+        { code: 7, label: "Direct access hostel" },
+        { code: 10, label: "Hospital" },
+        { code: 13, label: "Children’s home or foster care" },
+        { code: 14, label: "Bed and breakfast" },
+        { code: 19, label: "Rough sleeping" },
+        { code: 23, label: "Mobile home or caravan" },
+        { code: 24, label: "Home Office Asylum Support" },
+        { code: 25, label: "Any other accommodation" },
+        { code: 26, label: "Owner occupation (private)" },
+        { code: 28, label: "Living with friends or family" },
+        { code: 29, label: "Prison or approved probation hostel" },
+      ].each do |prevten|
```

Comment:
Thought since I was needing to update this (having stopped using the fake 2021 form) I may as well make it test what it says it's testing.